Engine/Graphics/DecalBuilder.cpp

```diff
@@ -57,19 +60,26 @@ void DecalBuilder::AddBloodsplat(float x, float y, float z, float r, float g,
 //----- (0049B525) --------------------------------------------------------
 void DecalBuilder::Reset(bool bPreserveBloodsplats) {
     if (!bPreserveBloodsplats) {
-        bloodsplat_container->std__vector_pBloodsplats_size = 0;
         bloodsplat_container->uNumBloodsplats = 0;
     }
     DecalsCount = 0;
 }
 
 //----- (0049B540) --------------------------------------------------------
-char DecalBuilder::ApplyDecals(int light_level, char LocationFlags, stru154 *a4, int a5,
-                               RenderVertexSoft *a6, IndoorCameraD3D_Vec4 *a7,
-                               char ClipFlags, unsigned int uSectorID) {
+char DecalBuilder::ApplyDecals(int light_level, char LocationFlags, stru154* a4, int a5,
+    RenderVertexSoft* a6, IndoorCameraD3D_Vec4* a7,
+    char ClipFlags, unsigned int uSectorID) {
+
+    // actually build decal geom and apply it
```

Comment:
can we rename to BuildAndApplyDecals or something then? i mean why comment if that could be a function name as easily - i'm all for descriptive function names